prison-mines/src/main/java/tech/mcprison/prison/mines/managers/MineManager.java

```diff
@@ -47,310 +47,34 @@
     int resetCount = 0;
 
     // NPE
-    private HashMap<UUID, LegacyMineManager> players;
+    private HashMap<UUID, MineManager> players;
```

Comment:
Why are you creating a whole mine manager for every player? It's probably inefficient, and managers are only supposed to have a single instance during execution.